Route/PathInfoBuilderSlashes.php

```diff
@@ -26,11 +26,12 @@ public function build(AdminInterface $admin, RouteCollection $collection)
     {
         $collection->add('list');
         $collection->add('create');
-        $collection->add('batch', null, array(), array('_method' => 'POST'));
+        $collection->add('batch', null, array(), array(), array(), '', array(), array('POST'));
```

Comment:
i think this only works by accident because php allows extra parameters and because by default a route allows all methods. the signature of [RouteCollection::add in version 2](https://github.com/sonata-project/SonataAdminBundle/blob/2.x/Route/RouteCollection.php#L66) does not have the extra arguments like version 3 has. i think for correct BC we need a version check and duplicate the above line, until we drop support for sonata admin 2.*